### PR TITLE
Some PHPSC 16 Stuff

### PR DESCRIPTION
**Pull request by michaelcullum**
@asgrim 

I might suggest you do `git merge --squash michaelc/phpsouthcoast16`

### CODE REVIEW

File: templates/forms/_talk.twig [hunk 1]
```diff
@@ -56,6 +57,7 @@
                 <option value="entry" {% if level == 'entry' %}selected {% endif %}>Entry level</option>
                 <option value="mid" {% if level == 'mid' %}selected {% endif %}>Mid-level</option>
                 <option value="advanced" {% if level == 'advanced' %}selected {% endif %}>Advanced</option>
+                <option value="mixed" {% if level == 'mixed' %}selected {% endif %}>Soft Skills/All levels</option>
```
Comment (asgrim) [inline templates/forms/_talk.twig:60]: I don't think a separate category is required here, I think it's implied that if it's a soft talk, the category doesn't matter anyway

File: templates/package.twig [hunk 2]
```diff
@@ -9,14 +9,15 @@
...
-        Speakers will be provided with a projector, a wireless lapel microphone and a screen
+        Speakers will be provided with a projector, a microphone and a screen
         for their presentation (size depends on the room). Speakers should bring any equipment
         they need to connect to projectors (VGA). It is also suggested that you reduce your
         dependency on the in-house internet connection as much as possible.
     </p>
     <p>
         When the Call for Papers closes, the organizers will sort through the submissions, making
-        their selections. Speakers will be notified as soon as the selection process has been completed.
+        their selections. The first stage of the speaker selection process is entirely anonomized.
```
Comment (asgrim) [inline templates/package.twig:19]: Nice! We didn't really publicly make a big deal about this last year, I think it'd be prudent to make more of a point of this fact this time.

File: templates/forms/_talk.twig (hunk 1 above)
Comment (michaelcullum) [inline templates/forms/_talk.twig:60]: This is something I've heard people talk about a lot being a really odd question and not applicable. Your call of course but I think people will appreciate it.

File: templates/package.twig (hunk 2 above)
Comment (michaelcullum) [inline templates/package.twig:19]: Indeed. :)

**Comment by asgrim** [comment]
thanks @michaelcullum , deployed

**Pull request merged**